controls/combobox/commands.md

```diff
@@ -8,11 +8,15 @@ slug: combobox-commands
 
 # .NET MAUI ComboBox Commands
 
-The [.NET MAUI ComboBox]({%slug combobox-overview%}) provides the following commands:
+The [.NET MAUI ComboBox]({%slug combobox-overview%}) provides a few commands that will help you manipulate its selection.
+
+## SelectAll Command
 
 - `SelectAllCommand` (`ICommand`)&mdash;Selects all items from the source.
 
-> The `SelectAll` command can be used only when `SelectionMode` is `Multiple`. An exception will be thrown if the command is invoked in `Single` `SelectionMode`.
+> The `SelectAll` command can be used only when `SelectionMode` is `Multiple`. In case the command is invoked in `Single` `SelectionMode`, the ComboBox throws an exception.
```

Comment:
- Avoid the passive voice.
- Avoid the visual clutter caused by the close placement of different styles (headings, inline code, a note)&mdash;as the section contains only a single line of text (except the content of the note), you don't need to use the note style to grab the attention of the reader. 
```suggestion
To use the `SelectAll` command, set the selection mode to `Multiple`. If you invoke the command in `Single` selection mode, the ComboBox throws an exception.
```